{
    "body": "Unable to install `py-ipykernel@5.3.4` due to failure of `install_data` phase.\r\n\r\nUsing:\r\n* `spack@develop`\r\n* eca1370abc65341421597018cc5b702a3e5ab46e\r\n* `2020-11-18 8:37 AM PST`\r\n\r\n```\r\n$> spack install py-ipykernel@5.3.4\r\n...\r\n==> py-ipykernel: Executing phase: 'build'\r\n==> py-ipykernel: Executing phase: 'install'\r\n==> py-ipykernel: Executing phase: 'install_data'\r\n==> Error: TypeError: install_data() takes 1 positional argument but 3 were given\r\n/builds/uo/e4s/spack/lib/spack/spack/build_environment.py:863, in _setup_pkg_and_run:\r\n        860        tb_string = traceback.format_exc()\r\n        861\r\n        862        # build up some context from the offending package so we can\r\n  >>    863        # show that, too.\r\n        864        package_context = get_package_context(tb)\r\n        865\r\n        866        logfile = None\r\n...\r\n```\r\n\r\nIntroduced via commit 8b431d17742a43aeca06d897dd5889a58df0d259 ?\r\n\r\n@ax3l @adamjstewart @scottwittenburg ",
    "user": "eugeneswalker",
    "url": "https://api.github.com/repos/spack/spack/issues/19985",
    "updated_at": "2020-11-19 02:45:30",
    "created_at": "2020-11-18 21:25:20",
    "closed_at": "2020-11-19 02:45:30",
    "state": "closed",
    "title": "py-ipykernel: install_data phase fails",
    "number": 19985,
    "milestone": null,
    "labels": [
        "build-error",
        "ecp",
        "e4s"
    ],
    "id": 746017958,
    "html_url": "https://github.com/spack/spack/issues/19985",
    "assignees": [],
    "comments": 0
}